recipe/recipe.yaml

```diff
@@ -1,15 +1,15 @@
 schema_version: 1
 
 context:
-  version: 1.11.489
+  version: 1.11.510
```

Comment:
Please don't mix version bumps with migrations here. This just causes logjams, e.g. in arrow right now:
```
Could not solve for environment specs
The following packages are incompatible
├─ aws-crt-cpp =0.31.0 * is requested and can be installed;
└─ aws-sdk-cpp =1.11.489 * is not installable because it requires
   └─ aws-crt-cpp >=0.29.9,<0.29.10.0a0 *, which conflicts with any installable versions previously reported.
```

Reply:
This was required. The old `aws-sdk-cpp` could not be build against the new AWS pins. I will add a migration for the new `aws-sdk-cpp` now.

Reply:
I had to mix version bumps and pin updates along the whole stack for latest releases :(

Reply:
> I had to mix version bumps and pin updates along the whole stack for latest releases :(

That means you need to couple the migrations as well. We cannot have a separate aws_c_common and another separate aws_sdk_cpp one, because they'll always conflict either way.

The only solution is to put both version bumps in the same migrator. In fact, I'd recommend to put _all_ aws-* version bumps into _one_ migrator, which completely avoids logjams and also reduces the PR churn